tests/conformance/bindings/bindings.go

```diff
@@ -237,7 +237,7 @@ func ConformanceTests(t *testing.T, props map[string]string, inputBinding bindin
 			req.Operation = bindings.GetOperation
 			resp, err := outputBinding.Invoke(context.Background(), &req)
 			assert.NoError(t, err, "expected no error invoking output binding")
-			if createPerformed {
+			if createPerformed && resp != nil {
```

Comment:
Is this a legit behavior change? Can resp be nil even when create was performed?

Usually, changes in conformance tests need to be carefully reviewed to understand if the variation of the component is acceptable.

Reply:
I rememeber see the `resp` with nil, but I can't reproduce the issue any longer, so I'm removing that change.